src/main.cpp

```diff
@@ -186,6 +193,29 @@ void opcontrol()
 			rightSpeed = analogY;
 		}
 
+		if (master.get_digital(DIGITAL_A))
```

Comment:
It is useful to add comment above IF block describing what it is going to do, or what's the purpose of this code.
I.e.
// Conveyor up
// Conveyor down
// Conveyor stopped